Sources/Bridges/OTelSwiftLog/LogHandler.swift

```diff
@@ -0,0 +1,188 @@
+import Foundation
+import OpenTelemetryApi
+import OpenTelemetrySdk
+import Logging
+
+// let the bridgename be the url of the package?
+let bridgename: String = "OTelSwiftLog"
+let version: String = "1.0.0"
+
+public struct InstrumentationScope{
+    public var name: String
+    public var version: String?
+    public var eventDomain: String?
+    public var schemaUrl: String?
+    public var includeTraceContext: Bool?
+    public var attributes: [String: AttributeValue]?
+}
+
+// Define a custom log handler
+struct OTelLogHandler: LogHandler {
+    private var scope: InstrumentationScope // Property to store instrumentation scope name
+    private var loggerProvider : LoggerProvider  // Property to set LoggerProvider
+    private var logger: OpenTelemetryApi.Logger 
+
+    // Define the log level for this handler
+    private var _logLevel: Logging.Logger.Level = .info
+    public var logLevel: Logging.Logger.Level{
+        get {
+            return self._logLevel 
+        }
+        set {
+            self._logLevel = newValue
+        }
+    }
+
+    // Define metadata for this handler
+    public var metadata: Logging.Logger.Metadata = [:]
+    public subscript(metadataKey key: String) -> Logging.Logger.Metadata.Value? {
+        get {
+            return self.metadata[key]
+        }
+        set {
+            self.metadata[key] = newValue
+        }
+    }
+    
+    // should default logger provider be a noop? or the sdk implementation?
+    public init(scope: InstrumentationScope = InstrumentationScope(name: bridgename, version: version),
+        loggerProvider: LoggerProvider = OpenTelemetrySdk.LoggerProviderSdk()) {
+
+        self.scope = scope
+        self.loggerProvider = loggerProvider
+        let loggerBuilder = self.loggerProvider.loggerBuilder(instrumentationScopeName: scope.name)
+            .configure(with: scope)
+        self.logger = loggerBuilder.build()
+    }
+
+    public func log(level: Logging.Logger.Level,
```

Comment:
Shouldn't the level be optional, as there is a default one? Or should it be mandatory and remove the default one?

Reply:
The log level defined above is for the LogHandler whereas this is the log level with which the message was logged

```
var logger = Logger(label: "com.example.myapp")
logger.logLevel = .info  // the level of of the handler
logger.handler = otelLogHandler
logger.debug("This is a debug message")  // this log message will not be printed as .debug  is lower than .info

logger.info("Logging with additional metadata") // this will be printed
```

Reply:
Then this should be optional: `Logging.Logger.Level?`

Reply:
even if the user does not set it explicitly, a default value is used by `Logger` to determine the logging level - here
https://github.com/apple/swift-log/blob/5d66f7ba25daf4f94100e7022febf3c75e37a6c7/Sources/Logging/Logging.swift#L128